src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -102,7 +102,7 @@ static NamedTypeSymbol getTupleUnderlyingType(ImmutableArray<TypeWithAnnotations
             NamedTypeSymbol tupleCompatibleType,
             ImmutableArray<string?> elementNames = default,
             ImmutableArray<bool> errorPositions = default,
-            ImmutableArray<Location> elementLocations = default,
+            ImmutableArray<Location?> elementLocations = default,
```

Comment:
>ImmutableArray<Location?> elementLocations [](start = 12, length = 42)

what prompted that change? 
Do we need a test for `Compilation.CreateTupleTypeSymbol` passing null locations? (I couldn't find one)

Reply:
https://github.com/dotnet/roslyn/pull/40946#discussion_r366123920